src/jsonpatcherproxy.js

```diff
@@ -276,13 +297,12 @@ const JSONPatcherProxy = (function() {
       const message =
         "You're accessing an object that is detached from the observedObject tree, see https://github.com/Palindrom/JSONPatcherProxy#detached-objects";
 
-      treeMetadata.handler.set = (
+      treeMetadata.handler.get = (
```

Comment:
Why is this change needed?

Reply:
If you look at the `master` branch, you will see that by mistake the `set` handler was repeated twice in this file: https://github.com/Palindrom/JSONPatcherProxy/blob/a585de9f58322b1f7a770cea61f50878f26ef728/src/jsonpatcherproxy.js#L279-L294

I overlooked it in my refactor PR #36. If I saw it later, I would remove one of the `set` getters.

So the real change here was to add a new `get` handler, but I combined this with the removal of the redundant `set` handler.

Reply:
Then do we need a message for get? `get` would not trigger any patch anyway, even if it was attached to observed object.

Reply:
@warpech ping

Reply:
ok, my idea was simply wrong and would just introduce more complexity to previous buggy solution.